submissions/Vas-P/popup/css/style.css

```diff
@@ -0,0 +1,199 @@
+*{
```

Comment:
You can use:
https://www.npmjs.com/package/reset-css
or
https://necolas.github.io/normalize.css/
to reset browser styles as well.